Modules/Messages/src/Messages.Application/Messages/Commands/SendMessage/Handler.cs

```diff
@@ -64,7 +64,7 @@ private async Task<List<RecipientInformation>> ValidateRecipients(SendMessageCom
 
             if (relationshipBetweenSenderAndRecipient == null)
             {
-                _logger.LogInformation("Sending message aborted. There is no relationship between sender ({sender}) and recipient ({recipient}).", sender, recipientDto.Address);
+                _logger.LogInformation("Sending message aborted. There is no relationship between the sender and the recipient.");
```

Comment:
```suggestion
                _logger.LogInformation($"Sending message aborted. There is no relationship between the sender and the recipient at index {i}.");
```

Reply:
I added a counter here instead of converting `forEach` to `for`, hope it is ok.